Error building OFT on linux

I following the build instructions listed [here](https://github.com/OpenFUSIONToolkit/OpenFUSIONToolkit/wiki/Building-OFT-on-Linux), but I am getting an error while making.

I successfully got through the Building External Libraries section without any errors being thrown.

I called `bash config_cmake.sh` without any errors, however it could not find some items:
```
-- Could NOT find OFT_OpenNURBS (missing: OpenNURBS_INCLUDE_DIR OpenNURBS_LIBRARY) 
-- Could NOT find OFT_SUPERLU (missing: SUPERLU_INCLUDE_DIR SUPERLU_LIBRARY) 
-- Could NOT find OFT_SUPERLU_DIST (missing: SUPERLU_DIST_INCLUDE_DIR SUPERLU_DIST_LIBRARY) 
```

The issue arises when I cd to the build_release directory and try to `make -j2`, I get the following error:
 ```
 258 |   CALL random_seed(put=oft_test_seed)
      |                       1
Error: Size of ‘put’ argument of ‘random_seed’ intrinsic at (1) too small (24/33)
make[2]: *** [base/CMakeFiles/oftbase.dir/build.make:129: base/CMakeFiles/oftbase.dir/oft_base.F90.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:449: base/CMakeFiles/oftbase.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```
I am using gcc version 9.4.0 and gfortran 9.4.0, running on Ubuntu 20.04.06
Any help would be appreciated.

--- a/src/base/oft_base.F90
+++ b/src/base/oft_base.F90
@@ -250,13 +250,15 @@ READ(io_unit,runtime_options,IOSTAT=ierr)
 CLOSE(io_unit)
 IF(ierr<0)CALL oft_abort('No runtime options found in input file.','oft_init',__FILE__)
 IF(ierr>0)CALL oft_abort('Error parsing runtime options.','oft_init',__FILE__)
-!---Seed pRNG if test run for repeatability
+!---Seed pRNG if test run for repeatability (does not work on < GCC 9)
+#if !defined(__GNUC__) || (__GNUC__ > 9)
 IF(test_run)THEN
   CALL random_seed(size=nargs)
   IF(nargs>SIZE(oft_test_seed))CALL oft_abort('pRNG seed size exceeds built in values', &
     'oft_init',__FILE__)
   CALL random_seed(put=oft_test_seed)
 END IF
+#endif
 !---Initialize PETSc or exit if requested but not available
 IF(use_petsc)THEN
 #ifdef HAVE_PETSC

